app/helpers/press_helper.rb

```diff
@@ -65,6 +65,12 @@ def google_analytics_url(subdomain)
     press.google_analytics_url.presence || parent_press(press)&.google_analytics_url
   end
 
+  def readership_map_url(subdomain)
+    press = Press.where(subdomain: subdomain)&.first
+    return if press.blank?
+    press.readership_map_url.presence || parent_press(press)&.readership_map_url
```

Comment:
Actually, I don't think this press helper method is actually called. Just grepping for readership_map_url it appears that all invocations are Press methods, i.e., @press.readership_map_url rather than readership_map_url('some_press'). The helper method I added early on, for completeness, and perhaps should have considered removing it when all is said and done.

So if I have the foregoing correct, then in fact I don't think we want the press helper pattern of behavior -- fall back to parent -- in this particular part of the app. We want the press at hand, or nothing at all. I think that is what the PR should be giving us as it stands now.